sympy/logic/boolalg.py

```diff
@@ -1533,6 +1533,10 @@ def _eval_rewrite_as_Piecewise(self, *args, **kwargs):
         from sympy.functions import Piecewise
         return Piecewise((args[1], args[0]), (args[2], True))
 
+    def as_set(self):
```

Comment:
This should probably be `_eval_as_set`, which then already exists a few lines up. Otherwise, I guess, expressions like `And(ITE,...))` will still be valid to perform `as_set` on but not `ITE.as_set()`. But then I also think that it makes sense to keep the functionality to rewrite `ITE`.